app/uk/gov/hmrc/apiplatformadminapi/models/ApplicationResponse.scala

```diff
@@ -20,7 +20,9 @@ import play.api.libs.json.{JsValue, Json, OFormat}
 
 import uk.gov.hmrc.apiplatform.modules.common.domain.models.{ApplicationId, Environment, LaxEmailAddress, UserId}
 
-case class User(userId: UserId, email: LaxEmailAddress, firstName: String, lastName: String)
+case class User(userId: UserId, email: LaxEmailAddress, firstName: String, lastName: String) {
```

Comment:
You could move `User` to a separate file if it seems appropriate now: it's no longer only used as part of `ApplicationResponse`.